dgbpy/dgbcombine.py

```diff
@@ -0,0 +1,131 @@
+    '''
+    Predict the data with trained multiple models.
+    
+    models = list of trained models
+    X = Inputs
+    model_type = classification or regression
+    '''
+    if model_type == modeltypes[0]:
+        return np.argmax((np.sum([models[i].predict_proba(X) for i in range(0, len(models))], axis=0)), axis=1)
+    elif model_type == modeltypes[1]:
+        return np.mean(np.column_stack([model.predict(X) for model in models]), axis=1)
+    
+def predict_onnx(sess, X, model_type=defmodeltype):
+    '''
+    Predict the data with trained multiple models that saved using onnx format.
+    
+    sess = onnx's session
+    X = Inputs
+    model_type = classification or regression
+    '''
+    if model_type == modeltypes[0]:
+        input_names = [i.name for i in sess.get_inputs()]
+        output_names = [o.name for o in sess.get_outputs()]
+        input_feed = {}
+        for i in input_names:
+            input_feed[i] = X.astype(np.float32)
+        outputs = sess.run(output_names, input_feed)
+        prob = []
+        for i in range(len(outputs)):
+            if i % 2 == 1:
+                prob_ = []
+                for out in outputs[i]:
+                    prob_.append(list(out.values()))
+                prob.append(prob_)
+        return np.argmax((np.sum(prob, axis=0)), axis=1)
+    elif model_type == modeltypes[1]:
+        input_names = [i.name for i in sess.get_inputs()]
+        output_names = [o.name for o in sess.get_outputs()]
+        input_feed = {}
+        for i in input_names:
+            input_feed[i] = X.astype(np.float32)
+        outputs = sess.run(output_names, input_feed)
+        return np.mean(np.column_stack(outputs), axis=1)   
+
+def save_model(models, outfnm, save_type=defsavetype, input_shape=None):
+    '''
+    Save the combined multiple models.
+    
+    models = list of trained models
+    outfnm = name for the saved models
+    save_type = type of saved models including, onnx, joblib, pickle, and .json for xgboost
+    input_shape = special for onnx type saved model
+    '''
+    if save_type == savetypes[0]:
+        sgs = []
+        for i,model in enumerate(models):
+            sgs.append(convert_sklearn(model, f"model{i}", initial_types=[(f"input_{i}", FloatTensorType([None, input_shape]))]))
+        init_comb_model = so.concat(sgs[0].graph, sgs[1].graph, rename_edges=True, rename_io=True)
+        combined_model = [init_comb_model]
+        for i in range(2, len(sgs)):
+            combined_model.append(so.concat(combined_model[i-2], sgs[i].graph, rename_nodes=True, rename_edges=False, rename_io=False))
+        so.graph_to_file(combined_model[-1], outfnm)
+    elif save_type == savetypes[1]:
+        return joblib.dump(models, open(outfnm, 'wb'), compress=1)
+    elif save_type == savetypes[2]:
+        return pickle.dump(models, open(outfnm, 'wb'))
+    else:
+        return pickle.dump(models, open(outfnm, 'wb'))
+
+def load_model(modelfnm, save_type=defsavetype):
```

Comment:
I am not sure if this function does anything different from dgbpy.dgbscikit.load, so it can probably be removed.